config/config_r12.js

```diff
@@ -0,0 +1,236 @@
+      "tip": {
+    "variant": "chrom-pos-ref-alt (alt is effect allele) - hover over a variant to see its cluster plot",
+    "consequence": "most severe consequence from Variant Effect Predictor",
+    "resources": "links to FinnGen browser, gnomAD and Open Targets",
+    "INFO": "imputation INFO score<br/>NA if the variant is not in the imputation panel",
+    "MAF": "minor allele frequency<br/>from imputed data if the variant is in the imputation panel<br/>from chip data otherwise",
+    "FIN enr.": "Finnish enrichment: AF_FIN / AF_NFSEE from gnomAD exomes 2.1.1<br/>(NFSEE=non-Finnish-Swedish-Estonian European)<br/>NA if there are neither FIN nor NFSEE alleles in gnomAD or the variant is not in gnomAD",
+    "p-val": "additive model p-value from imputed data",
+    "p-val rec": "recessive model p-value from imputed data",
+    "p-val chip": "additive model p-value from chip genotype data",
+    "beta": "additive model effect size beta from imputed data",
+    "beta rec": "recessive model effect size beta from imputed data",
+    "beta chip": "additive model effect size beta from chip genotype data",
+    "rec-add": "recessive -log10(p) minus additive -log10(p)<br/>positive values mean recessive signal is more significant",
+    "leads": "data on possible more significant non-coding lead variants in the region<br/>if the lead variant is more than two orders of magnitude stronger than the coding variant, the column is yellow"
+  }
+} };
+// configuration
+const userInterface = {
+	/*
+    lof : {
+	table : { columns : [
+	    { type : 'lofPhenotype' },
+	    { type : 'lofGene' },	    
+	    { type : 'pValue' , attributes : { accessor: 'p_value' } },
+	    { type : 'variantList' },
+	    { type : 'beta' },
+	] }
+
+    },  
+  notFound: {
+    entity: { message: notFoundEntityMessage },
+    page: { message: notFoundPageMessage }
+  }, */
+  coding: coding,
+  about: { banner: aboutBanner },
+    variant : { table : { columns : variantTableColumns , defaultSorted : [{
+  id: 'mlogp',
+  desc: true
+}] } }, 
+	gene: { 
+	    lossOfFunction: { tableColumns :  [{ type : "phenotype" },
+					       { type : "variants" , attributes : { minWidth : 400 } },
+					       { type : "pValue" },
+					       { type : "or" } ] },
+	    lz_config : { ld_panel_version : "sisu42" } ,
+	     pqtlColocalizations : { tableColumns : genePqtlTableColumns }  },
+  phenotype: {
```

Comment:
Could you please add the following to the phenotype config: `r2_to_lead_threshold: 0.6`

Reply:
done in [6210856](https://github.com/FINNGEN/pheweb/pull/438/commits/6210856d201b5a2d5d9a21068913fc1ddebf537c)